src/app/models/evt-models.ts

```diff
@@ -126,3 +126,9 @@ export interface WitnessGroup {
     witnesses: string[];
     groupId: string;
 }
+
+export interface GridItem{
```

Comment:
Add space between interface name and ”{”

Reply:
Or use autoindent!